src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -1222,4 +1222,116 @@ I agree to all of the foregoing:</value>
   <data name="Rename_0_to_1" xml:space="preserve">
     <value>Rename {0} to {1}</value>
   </data>
+  <data name="NamingSpecification_CSharp_Class" xml:space="preserve">
+    <value>class</value>
+    <comment>This string can be found under "Tools | Options | Text Editor | C# | Code Style | Naming | Manage Specifications | + | Symbol kinds". All of the "NamingSpecification_CSharp_*" strings represent language constructs, and some of them are also actual keywords (including this one).</comment>
+  </data>
```

Comment:
If you want to force this to have the text "class" in _all_ locales, you can put "{Locked}" at the beginning of the `<comment>` and the translator will know not to touch it. Though at that point you're better off just hard-coding it in the source and not bothering with a resource. It's not clear to me if you want this to be "class" regardless of locale, or if you want it translated.

Reply:
I've marked things as {Locked} when they are an exact language keyword.